Toolset/libraries/revcommonlibrary.livecodescript

```diff
@@ -133,11 +133,6 @@ on revSetWindowsShellCommand
   if the platform is not "Win32" then exit revSetWindowsShellCommand
   set the hideConsoleWindows to true
   if $COMSPEC is not empty then set the shellCommand to $COMSPEC
```

Comment:
This can just be:
```
set the shellCommand to $COMSPEC
```
Since we are making it an error to try and `shell()` if shellCommand is empty.